src/database/migrations/1745051642191-create-college-table.ts

```diff
@@ -0,0 +1,23 @@
+import { MigrationInterface, QueryRunner } from "typeorm";
+
+export class CreateCollegeTable1745051642191 
+implements MigrationInterface {
+    name = 'CreateCollegeTable1745051642191'
+
+    public async up(queryRunner: QueryRunner): Promise<void> {
+        await queryRunner.query(`CREATE TABLE "college" (
+            "id" SERIAL PRIMARY KEY, 
+            "name" VARCHAR(255) NOT NULL, 
+            "uniform" VARCHAR(255), 
+            "provost" VARCHAR(255), 
+            "year" INTEGER, 
```

Comment:
I dont understand what sort of data year column is to store

If it's the year it was established. Then for the love of God, use a better descriptive variable. And at that, should be a date column not integer